src/components/HomeHeader.js

```diff
@@ -61,6 +61,11 @@ class HomeHeader extends React.Component {
               <h3 className="page-title">Litmus</h3>
             </div>
           </a>
+
+          <Link to={process.env.PUBLIC_URL} className="home">
+          <h3 className="top-header-home">Home</h3>
```

Comment:
can you use home icon from here
`https://react-icons.netlify.com/#/icons/fa`

Reply:
@GavinAf, can please resolve this comment

Reply:
@inyee786, you didn't specify if you wanted it added to the left or replace, so I replaced it. Let me know if you want it the other way.